tests/test_loggers/test_console_logger.py

```diff
@@ -31,6 +31,7 @@ def check_logged_transforms(log_file: str, transforms: Sequence[Transform]):
             assert transform.__class__.__name__ in line
 
 
+@pytest.mark.xfail(reason="TSDataset 2.0: blocked by another transform")
```

Comment:
Mark tests for loggers with "skip" to avoid problems with `tslogger` object

Reply:
In fact it is better to add tests from `test_logger` folder [here](https://github.com/tinkoff-ai/etna/blob/645c71a5290d2cd9de7a8d12a6450ab44ae8b86c/tests/conftest.py#L12)